Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -288,7 +294,11 @@ import Foundation
     @objc(POST:parameters:httpType:completion:)
     public func post(_ path: String, parameters: [String: Any]? = nil, httpType: BTAPIClientHTTPService, completion: @escaping RequestCompletion) {
         fetchOrReturnRemoteConfiguration { [weak self] configuration, error in
-            guard let self else { return }
+            guard let self else {
+                completion(nil, nil, BTAPIClientError.deallocated)
+
```

Comment:
```suggestion
```

Reply:
Updated as part of this change: https://github.com/braintree/braintree_ios/pull/1049/commits/1fadabdbc626829f4dbb6e6decf9fb59066acf2a